specs/041-codex-deep-review/data-model.md

```
@@ -0,0 +1,73 @@
+  copilot: false      # existing
+  codex: true         # NEW: enabled by default
+```
+
+**Attributes**:
+- `codex` (boolean, default: `true`): Whether to detect and invoke Codex CLI during deep review
+
+### Finding Schema (unchanged, new source_agent value)
+
+The common finding schema is reused with a new `source_agent` value:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)